pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -510,38 +498,32 @@ class HassTypeHintChecker(BaseChecker):  # type: ignore[misc]
 
     def __init__(self, linter: PyLinter | None = None) -> None:
         super().__init__(linter)
-        self.current_package: str | None = None
-        self.module: str | None = None
+        self._function_matchers: list[TypeHintMatch] = []
 
     def visit_module(self, node: astroid.Module) -> None:
```

Comment:
In `pylint` and `astroid` we usually don't use the toplevel import for nodes.
Instead we do `from astroid import nodes` and `nodes.Module`, etc. Of course that's just a convention we use, nothing more.

Reply:
I think I'll do that in a follow-up PR. This one is big enough :)